File: Java/608.java
import java.util.Scanner;

/**
 *
 * 608 - Peligro por Hielo - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cantidad = sc.nextInt();
        while (cantidad != 0) {
            int alarma = 0;
            boolean distancia = false;
            for (int i = 1; i <= cantidad; i++) {
                int grados = sc.nextInt();
                if (grados <= 4 && !distancia) {
                    alarma++;
                    distancia = true;
                } else if (grados > 6) {
                    distancia = false;
                }
            }
            System.out.println(alarma);
            alarma = 0;
            distancia = false;
            cantidad = sc.nextInt();
        }
    }
}

File: C++/608.cpp
#include <iostream>

using namespace std;

// 608 - Peligro por Hielo - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy

int main()
{
    int loop = 0;
    cin >> loop;
    while (loop != 0) {
        int degrees = 0;
        bool frozen = false;
        int alarm = 0;
        for (int i = 0; i<loop; i++) {
            cin >> degrees;
            if (!frozen && degrees <= 4) {
                alarm++;
                frozen = true;
            } else if (frozen && degrees > 6) {
                frozen = false;
            }
        }
        cout << alarm << endl;
        cin >> loop;
    }
    return 0;
}
